dotCMS/src/main/java/com/dotmarketing/db/DbConnectionFactory.java

```diff
@@ -438,6 +438,38 @@ public static boolean isDBTrue(String value) {
 
     }
 
+    /**
+     * Return true if the String represent one on the false
+     * boolean value for the current database
+     * @param value String representation of the boolean
+     * @return true is the string represents a DB false value
+     */
+    public static boolean isDBFalse(final String value) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 5, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1142)